nestjs/src/gratitudes/dto/create-gratitude.dto.ts

```diff
@@ -1,12 +1,12 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsEnum, IsNotEmpty, IsNumber, IsString, MinLength } from 'class-validator';
+import { IsArray, IsEnum, IsNotEmpty, IsNumber, IsString, MinLength } from 'class-validator';
 import { Badge } from './badge.dto';
 
 export class CreateGratitudeDto {
   @IsNotEmpty()
-  @IsNumber()
-  @ApiProperty()
-  userId: number;
+  @IsArray()
+  @ApiProperty({ type: [Number] })
+  userIds: number[];
```

Comment:
not sure whether I should update or create new DTO for handling gratitudes in a batch, but seems like this DTO is being used only once in the application - to create the single gratitudes - so I decided to update the existing DTO, rather than creating a new one